templates/leaderboard.html

```diff
@@ -36,7 +36,7 @@ <h6 v-if="available">
                             <strong>[[prompt.start]]</strong> to <strong v-if="runs.length > 0">[[prompt.end]]</strong><strong v-else>...</strong>
                         </h6>
                         
-                        <small> Prompt created by: 
+                        <small v-if="!lobbyId"> Prompt created by: 
```

Comment:
I think this was added in #510?

Reply:
Yeah, this will merge cleanly so I think it's fine